src/llmcompressor/observers/base.py

```diff
@@ -52,12 +51,12 @@ def forward(
         :return: tuple of scale and zero point based on last observed value
         """
         self.record_observed_tokens(observed)
+        if should_calculate_gparam:
+            return self.get_gparam(observed=observed)
```

Comment:
`get_gparam` returns a single Tensor, but the return signature here is `Tuple[FloatTensor, IntTensor]`. Can this be set up in a way where function signatures do not change? Is the `calculate_gparam` method entirely separable from the qparams method?

Reply:
Yeah they're returning two separate sets of parameters but use the same underlying min/max tacking that the observers provide

Reply:
No but I think the pattern is to use the singular forward method for all observer calls.

Reply:
ok, at the very least we should update the signature of `forward` to `-> Union[FloatTensor, Tuple[FloatTensor, IntTensor]]`

Reply:
Yeah will update. I'm ok with separating out the functionality but that would mean moving away from the forward design